src/ucp/core/ucp_ep.c

```diff
@@ -2382,6 +2382,15 @@ int ucp_ep_config_test_rndv_support(const ucp_ep_config_t *config)
            (config->key.cm_lane  != UCP_NULL_LANE);
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_ep_is_am_keepalive(ucp_ep_h ep, ucp_lane_index_t lane)
+{
+    return (ep->flags & UCP_EP_FLAG_REMOTE_ID) && /* remote ID defined */
```

Comment:
pls add a comment that if we have 2iface transport we need to send an active-message based keepalive to check the remote endpoint still exists